### PR TITLE
fix : resolve tracking GA option inside using tui component

### PR DESCRIPTION
**Pull request by jungeun-cho**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: src/js/factory/calendar.js [hunk 1]
```diff
@@ -509,11 +509,11 @@ var mmin = Math.min;
  * });
  */
 function Calendar(container, options) {
-    var opt = util.extend({
+    options = util.extend({
```
Comment (dongsik-yoo) [inline src/js/factory/calendar.js]: `options`가  null일 때 확인 필요

File: src/js/view/popup/scheduleCreationPopup.js [hunk 2]
```diff
@@ -24,7 +24,7 @@ var ARROW_WIDTH_HALF = 8;
  * @param {HTMLElement} container - container element
  * @param {Array.<Calendar>} calendars - calendar list used to create new schedule
```
Comment (dongsik-yoo) [inline src/js/view/popup/scheduleCreationPopup.js]: param 주석 추가 필요

File: src/js/factory/calendar.js (hunk 1 above)
Comment (jungeun-cho) [reply src/js/factory/calendar.js]: `{usageStatistics: true}` 정상동작합니다.

File: src/js/view/popup/scheduleCreationPopup.js (hunk 2 above)
Comment (jungeun-cho) [reply src/js/view/popup/scheduleCreationPopup.js]: 주석 추가했습니다.

**Pull request merged**